src/statuspage.test.ts

```diff
@@ -0,0 +1,32 @@
+import DefaultOperations from './defaultOperations';
+import StatusPage from './statuspage';
+
+const pageId = "12345";
+const apiKey = "##key##";
+
+test('I can import StatusPage', () => {
+    expect(StatusPage).toBeDefined();
+});
+
+test('I can require StatusPage', () => {
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/zipper-team/statuspage/pullRequest?prid=7096534)
```suggestion
test("I can require StatusPage", () => {
```